inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -616,11 +616,26 @@ def normalize_journal_title_entry(publication_info):
         RecordMetadata.json['_collections'].op('?')('Journals')).filter(
         cast(RecordMetadata.json['short_title'], String) == type_coerce(normalized_title, JSON))
     result = db.session.execute(ref_query).fetchone()
-    journal_record = result.records_metadata_json['self'] if result else None
+    journal_data = result.records_metadata_json if result else None
+
+    if not journal_data:
+        return
+
+    journal_record = journal_data['self']
 
     if journal_record:
         publication_info['journal_record'] = journal_record
 
+    if not check_mark(obj, 'is-update') and journal_data.get('inspire_categories'):
```

Comment:
do we need the first check? we can always write the categories, even if we don't need them later.